modules/scala-3-step-filter/src/main/scala/ch/epfl/scala/debugadapter/internal/stepfilter/ScalaStepFilterBridge.scala

```diff
@@ -110,11 +119,30 @@ class ScalaStepFilterBridge(
           case _ => { false }
       }
       case _ => false
+  def isAndOrFunction(tpe: Type): Boolean =
```

Comment:
Why do we have `Function` in the method name?

```suggestion
  def isAndOrOr(tpe: Type): Boolean =
```